RELEASE_PLAN.md

```diff
@@ -0,0 +1,58 @@
+## Release Plan
+
+Starting with version `v2.4`, the Nuxt core team will adhere to a formalized release plan (as good as possible).
+Also, an end of life for older major versions is defined with this document
+
+### Major versions (3.x.y, 4.x.y, ...)
+
+We aim for a major release once every **6 months**. This depends on a few factors though:
```

Comment:
We never discussed definite time boxes for a major release besides the aim for once every 6 months.

If there are no breaking changes for example, we can't and shouldn't release a new major. Another exception would be unexpected major updates in important dependencies.

Not sure if we should give a date range.

Reply:
Thanks for explanation @manniL.

> If there are no breaking changes, for example, we can't and shouldn't release a new major. Another exception would be unexpected major updates in important dependencies.

This could be really good to mentioned somehow in the text. Was little confusing at least to me at first.

Reply:
@pi0 You mean as addition to

> * Major version releases of critical dependencies like Vue, Webpack or Babel
> * Number of breaking changes "in the queue"

Reply:
@pi0 Do you think it's better explained now? :relaxed:

Reply:
Definitely. Thanks :+1: